### DISCUSSION TITLE
Storage folder by default? I forgot to configure .yaml

### DISCUSSION THREAD

**Discussion by starbotica** [photoprism/photoprism#3817] [Q&A]
Hi! I hope you can help a newbie. I skipped the step where the documentation clearly states: 

"a storage folder mount must always be configured in your docker-compose.yml file so that you do not lose these files after a restart or upgrade"

and now I am not able to find the files in the server.

If I delete photoprism and restart the process, will this folder be deleted and the storage will be available? Thank you in advance.

**Comment by lastzero** [comment]
Indeed, the files are then inside the container on temporary storage that will be lost when you restart (or upgrade) the container. However, you can use the `docker compose cp` command to copy files from `/photoprism/storage` inside the `photoprism` service container to a host directory:
- https://docs.docker.com/engine/reference/commandline/compose_cp/
